tests/test_constraints.py

```diff
@@ -127,7 +127,7 @@ def comp_constraints(solver):
                            Numexpr != Constant             (CPMpy class 'Comparison')
         - Numeric inequality (>=,>,<,<=): Numexpr >=< Var  (CPMpy class 'Comparison')
     """
-    for comp_name in Comparison.allowed:
+    for comp_name in sorted(Comparison.allowed):
```

Comment:
I guess we need to check everywhere that we don't loop over unordered collections?
there is also Operator.allowed for example (probably those are the only 2 cases)

Reply:
Actually, I rememberd that this works because `Operator.allowed` is `dict` (while `Comparator.allowed` is a `set`), and `dict`'s nowawadays preserve insertion order when iterating (https://docs.python.org/3.7/library/stdtypes.html#mapping-types-dict) since Python 3.7.

What is our minimally supported python version? `setup.py` has `>=3.6`? In that case, I should add `sorted`.

Reply:
I guess this might be a reason to make that >= 3.7 ?